extensions/gdata/src/com/google/refine/extension/gdata/GDataImportingController.java

```diff
@@ -197,95 +168,56 @@ private void listFusionTables(Fusiontables service, JSONWriter writer)
     private void doInitializeParserUI(
         HttpServletRequest request, HttpServletResponse response, Properties parameters)
             throws ServletException, IOException {
-        
         String token = TokenCookie.getToken(request);
-        
         String type = parameters.getProperty("docType");
         String urlString = parameters.getProperty("docUrl");
+        JSONObject result = new JSONObject();
+        JSONObject options = new JSONObject();
         
-        URL url = new URL(urlString);
-        try {
-            JSONObject result = new JSONObject();
-            JSONObject options = new JSONObject();
-            JSONUtilities.safePut(result, "status", "ok");
-            JSONUtilities.safePut(result, "options", options);
-            
-            JSONUtilities.safePut(options, "skipDataLines", 0); // number of initial data lines to skip
-            JSONUtilities.safePut(options, "storeBlankRows", true);
-            JSONUtilities.safePut(options, "storeBlankCellsAsNulls", true);
+        JSONUtilities.safePut(result, "status", "ok");
```

Comment:
@jackyq2015 nothing about encoding options ? needed / supported ?

Reply:
There is no encoding options to get a spreadsheet:
https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/get

I think it make sense because all the data can be stored and unified to utf-8 by google service/google API.